README.rst

```diff
@@ -31,19 +31,22 @@ Crate JDBC driver   Crate
 ``1.11.X``          ``0.54.X``
 ``1.12.X``          ``0.54.X``
 ``1.13.X``          ``0.55.X``
+``2.0.X``           ``0.57.X``
 =================== ==========
 
 Currently we don't provide the testing version of the Crate JDBC driver.
 Therefore, using the driver with the latest testing releases of the Crate
 might not be possible due compatibility issues. As a workaround, you can build
```

Comment:
Starting with 2.X the compatibility issues should be gone - that's one of the major reason we're doing this after all